README.md

```diff
@@ -3,9 +3,7 @@
 <!-- User badges on first line (language docs & chat) -->
 [![Language Docs](https://img.shields.io/badge/DOCS-LANGUAGE-blue?style=for-the-badge)](https://prql-lang.org)
 [![Discord](https://img.shields.io/discord/936728116712316989?label=discord%20chat&style=for-the-badge)](https://discord.gg/eQcfaCmsNc)
-[![VSCode](https://img.shields.io/visual-studio-marketplace/v/prql.prql?label=vscode&style=for-the-badge)](https://marketplace.visualstudio.com/items?itemName=prql.prql)
```

Comment:
Is it that you think we're over-badged here? I think the VSCode plugin is good and it's useful to show that we're catering to developers too. Ironically we have better VSCode support than our friends at dbt!

Reply:
It's a bit crowded I think. We can have vscode plugin under integrations.

Reply:
OK